apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateCopyJobOptions.cs

```diff
@@ -43,6 +44,11 @@ public sealed class CreateCopyJobOptions : JobCreationOptions
         /// </summary>
         public CopyOperationType? OperationType { get; set; }
 
+        /// <summary>
+        /// Delegate that is used to modify the underlying <see cref="JobConfigurationTableCopy"/>.
```

Comment:
Let's get everything else in the PR ready, then I'll have a go at wordsmithing this. (I owe the team a doc about XML documentation comments...)